client/packages/react-native/src/version.js

```diff
@@ -0,0 +1,3 @@
+const version = "v0.14.27-dev";
```

Comment:
nit: should we add a comment here saying it's auto-generated?

Reply:
Good idea!